modules/47_regipol/regiCarbonPrice/postsolve.gms

```diff
@@ -256,18 +256,29 @@ loop((ttot,ttot2,ext_regi,emiMktExt,target_type_47,emi_type_47)$pm_emiMktTarget(
       loop(iteration2$((iteration2.val le iteration.val) and (iteration2.val eq p47_slopeReferenceIteration_iter(iteration,ttot2,ext_regi))), !!reference iteration for slope calculation
 ***     if it is the first iteration or the reference iteration changed, initialize the rescale factor based on remaining deviation
         if(iteration.val eq p47_slopeReferenceIteration_iter(iteration,ttot,ext_regi),
-          pm_factorRescaleemiMktCO2Tax(ttot,ttot2,ext_regi,emiMktExt) = (1+pm_emiMktTarget_dev(ttot,ttot2,ext_regi,emiMktExt)) ** 2;
+          pm_factorRescaleemiMktCO2Tax(ttot,ttot2,ext_regi,emiMktExt) = power(1+pm_emiMktTarget_dev(ttot,ttot2,ext_regi,emiMktExt), 2);
```

Comment:
Power works on exponentials for both negative and positive numbers, meanwhile `**` is not defined for negative numbers. 
see the attention block here: https://www.gams.com/latest/docs/UG_Parameters.html#UG_Parameters_StandardArithmeticOperations

So it would be theoretically possible, although I never found this case, that for REMIND negative emission values we would have a non-defined error in the rescale factor calculation.

Reply:
thanks for the explanation!